lack/logger_manager.py

```diff
@@ -1,13 +1,20 @@
 from .logger import *
 
+
 class LoggerManager:
     DEBUG = logging.DEBUG
     INFO = logging.INFO
     CRITICAL = logging.CRITICAL
     EXCEPTION = logging.ERROR
 
+    main_logger_name = "dynamo"
```

Comment:
you are still using 'dynamo". Can we have an argument to pass str (the string can be either "dynamo" or dynode, etc?) If you have a better idea, let me know